### PR TITLE
docs: Create Fortnightly Dev Call.ics

### PR DESCRIPTION
**Pull request by richb-hanover**
This file creates events every 2nd and 4th Sunday between January and March 2023. Update it periodically (edit the "UNTIL=" date) as needed

### CODE REVIEW

**Comment by max-sixty** [comment]
Super! Shall we link to this from `CONTRIBUTING.md`?

**Comment by richb-hanover** [comment]
Good thought about updating CONTRIBUTING.md. I need to horse around to make the .ics file clickable before merging

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -11,6 +11,9 @@ ways to start:
 - Find an issue labeled
   [Good First Issue](https://github.com/prql/prql/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22)[^1]
   and start contributing to the code.
+- Join our [regular Develper Call.](https://github.com/PRQL/prql/issues/1083)
```
Comment (max-sixty) [inline CONTRIBUTING.md]:
```suggestion
- Join our [regular Developer Call.](https://github.com/PRQL/prql/issues/1083)
```

**Comment by max-sixty** [comment]
Super! Good place to link from.
Thanks @richb-hanover

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -11,6 +11,9 @@ ways to start:
 - Find an issue labeled
   [Good First Issue](https://github.com/prql/prql/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22)[^1]
   and start contributing to the code.
+- Join our [regular Developer Call.](https://github.com/PRQL/prql/issues/1083)
+  Here's an
+  [iCal file to add the dates to your calendar.](https://raw.githubusercontent.com/prql/prql/main/Fortnightly_Dev_Call.ics)
```
Comment (max-sixty) [inline CONTRIBUTING.md]:
```suggestion
  [iCal file to add the dates to your calendar.]
```

Going to merge this and then add this back to get around the link check

**Pull request merged**